backend/data/schema/FormFields.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+const uniqueValidator = require('mongoose-unique-validator');
+
+const Schema = mongoose.Schema;
+
+const FormFieldsSchema = new Schema({ 
```

Comment:
A new collection (mongodb table) I introduced which will store all the form sections/questions that will be loaded.

The term update form should be customizable (ie. admins can add new questions, modify the selectable options per question, etc.), so we are loading all these form questions from the back end.